src/pages/home/sidebar/SidebarLinks.tsx

```diff
@@ -16,37 +15,33 @@ import KeyboardShortcut from '@libs/KeyboardShortcut';
 import Navigation from '@libs/Navigation/Navigation';
 import onyxSubscribe from '@libs/onyxSubscribe';
 import * as ReportActionContextMenu from '@pages/home/report/ContextMenu/ReportActionContextMenu';
-import safeAreaInsetPropTypes from '@pages/safeAreaInsetPropTypes';
 import * as App from '@userActions/App';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import type {Modal, Policy, Report} from '@src/types/onyx';
 
-const basePropTypes = {
-    /** Toggles the navigation menu open and closed */
-    onLinkClick: PropTypes.func.isRequired,
-
-    /** Safe area insets required for mobile devices margins */
-    insets: safeAreaInsetPropTypes.isRequired,
+type SidebarLinksOnyxProps = {
+    activePolicy: OnyxEntry<Policy>;
 };
 
-const propTypes = {
-    ...basePropTypes,
-
-    optionListItems: PropTypes.arrayOf(PropTypes.string).isRequired,
-
-    isLoading: PropTypes.bool.isRequired,
-
-    // eslint-disable-next-line react/require-default-props
-    priorityMode: PropTypes.oneOf(_.values(CONST.PRIORITY_MODE)),
-
-    isActiveReport: PropTypes.func.isRequired,
+type SidebarLinksProps = SidebarLinksOnyxProps & {
+    onLinkClick: () => void;
+    insets: EdgeInsets | undefined;
```

Comment:
```suggestion
    insets?: EdgeInsets;
```

Reply:
I think in that case it will be better to leave it as it is because we always want that prop to be passed to the component but the value can be undefined

Reply:
I understand this logic for functions, but not sure about components. If the component is working ok if the param is undefined then maybe it makes sense to make it optional. But I'm okay to keep it as is if it's for consistency with other places 👌